### ISSUE TITLE
Even number pixel ellipses with ctrl and alt

### ISSUE THREAD

**Issue by SilverDarkBlade** [aseprite/aseprite#3559]
When using the ellipse tool with CTRL or after changing the angle with ALT you are unable to make ellipses that are an even number of pixels wide or tall.

**Comment by martincapello** [comment]
Hey there! I tested this and was able to reproduce it. However we will need a bit of discussion here, because it works this way by design I think. 

When you use CTRL or rotate the ellipse using ALT, Aseprite tries to center it around one pixel, then the only way to keep the ellipse accurately centered is by using odd sizes, because otherwise you end up by having one pixel more at one side. 

@dacap we need you opinion here.

**Comment by SilverDarkBlade** [comment]
Yes I think it would be great if you could decide to center it around/in the middle of a group of 4 pixels instead. 

Currently I haven’t been able to get any ellipse with alt to line up with the specific isometric grid I am using because it is an even number tile size.
